internal/handlers/common/ismaster.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+	"time"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// IsMaster is a common implementation of the isMaster command used by deprecated OP_QUERY message.
+func IsMaster(ctx context.Context, _ *wire.OpQuery) (*wire.OpReply, error) {
```

Comment:
```suggestion
func IsMaster(context.Context, *wire.OpQuery) (*wire.OpReply, error) {
```


Quite a minor comment, but `ctx` doesn't seem to be used, right?

Reply:
Not at the moment but shouldn't we keep it as it may contain connection information in the future? The `isMaster` command can also contain the optional field `saslSupportedMechs` which may be useful. Should I have not discarded the query?